SharpLink/LavalinkManager.cs

```diff
@@ -97,6 +97,13 @@ private void SetupManager()
                             players.Remove(oldVoiceState.VoiceChannel.Guild.Id);
                         }
                     }
+                    else if (oldVoiceState.VoiceChannel != null && newVoiceState.VoiceChannel != null && oldVoiceState.VoiceChannel.Id != newVoiceState.VoiceChannel.Id && players[newVoiceState.VoiceChannel.Guild.Id] != null)
+                    {
+                        logger.Log($"VOICE_STATE_UPDATE({newVoiceState.VoiceChannel.Guild.Id}, Moved)", LogSeverity.Debug);
+
+                        // Moved
+                        players[newVoiceState.VoiceChannel.Guild.Id].VoiceChannel = newVoiceState.VoiceChannel;

```

Comment:
Can we change this to perform similar to the above with `UpdateSessionAsync` with a session change of `Moved`? While this won't particularly interact with Lavalink consistency with the other two events is a bit better

Reply:
I created a new enum called `SessionChange.Moved` because I was unsure about using `SessionChange.MoveNode` even though it is never actually used in SharpLink. My instinct was just to avoid it and create a new value not knowing what it could be used for in the future.